model/flow/sealing_segment.go

```diff
@@ -87,6 +87,14 @@ func (segment *SealingSegment) Highest() *Block {
 	return segment.Blocks[len(segment.Blocks)-1]
 }
 
+// Lowest is the lowest block in the sealing segment by height.
+func (segment *SealingSegment) Lowest() *Block {
```

Comment:
My IDE shows it's not used anywhere. Do we need it? It seems strange to me that we provide lowest block in this way. I was under the impression that we provide extra blocks only for history reasons and for specialized cases but lowest block is actually first block of sealing segment(not extra blocks)

Reply:
Removed. I had this for a test case at one point but ended up doing it a different way